lib/timeline/Timeline.js

```diff
@@ -173,9 +174,19 @@ function Timeline (container, items, groups, options) {
 
         var start = me.options.start != undefined ? me.options.start : range.min;
         var end   = me.options.end   != undefined ? me.options.end   : range.max;
-        me.setWindow(start, end, {animation: false}, this._activateOnInitialDrawDone.bind(this));
+        me.setWindow(start, end, {animation: false});
       } else {
-        me.fit({animation: false}, this._activateOnInitialDrawDone.bind(this));
+        me.fit({animation: false});
+      }
+    }
+
+    if (!me.initialDrawDone && me.initialRangeChangeDone) {
```

Comment:
Ah yes right. If I was just a bit clearer in my head I should have remarked on that. Make sense, not directly connected to the while-loop.